src/Agent/CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Fixes Issue [#XX](https://github.com/newrelic/newrelic-dotnet-agent/issues/XX)
 
 ## [8.35] - 2020-11-09
 
+### New Features
+* **.NET 5 GA Support** <br/>
+We have validated that this version of the agent is compatible with .NET 5 GA. See the [compatibility and requirements for .NET Core](https://docs.newrelic.com/docs/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core) page for more details.
```

Comment:
Vu, 
Could you add,  
Moving forward, information on supported .NET versions will be reserved for the compatibility matrix alone.

Reply:
I like to keep it as it is for now. I know we already discussed this with the team about not mentioning it in the release note, but I now can see the benefit of having this information in the release note.